src/Test/UnitTests/attributes.jl

```diff
@@ -31,6 +31,27 @@ function silent(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["silent"] = silent
 
+"""
+    timelimitsec(model::MOI.ModelLike, config::TestConfig)
```

Comment:
The [style guide](http://www.juliaopt.org/JuMP.jl/v0.19.2/style/#Naming-1) says this should be named `time_limit_sec`.

Reply:
I'll make that change but I was wondering why JuMP style guide differ from [Julia style guide](https://docs.julialang.org/en/v1/manual/style-guide/index.html)?

Reply:
The julia style guide has an ambiguous guideline for underscores: "functions are lowercase (maximum, convert) and, when readable, with multiple words squashed together (isequal, haskey)." According to this rule, I could argue that `timelimitsec` is not readable. Someone else could argue that it is. As the preamble to the JuMP style guide states, a goal is "reducing the amount of time spent on bike-shedding by establishing basic naming and formatting conventions". By using underscores to separate words *always*, we don't have to spend time debating this.

Reply:
Ok, thank you very much for taking the time to answer